test1/styleblog.css

```diff
@@ -0,0 +1,482 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+@font-face{
+    font-family: GTZ;
+    src: url(img/font/GTZirkon-Book.woff2);
+}
+
+body{
+    font-family: GTZ;
+}
```

Comment:
You could create a `common.css` file and import that in all the 3 pages instead of copy pasting same style in each page.